flexmeasures/utils/app_utils.py

```diff
@@ -265,3 +276,26 @@ def register_plugins(app: Flask):
         app.config["LOADED_PLUGINS"][plugin_name] = plugin_version
     app.logger.info(f"Loaded plugins: {app.config['LOADED_PLUGINS']}")
     sentry_sdk.set_context("plugins", app.config.get("LOADED_PLUGINS", {}))
+
+
+def check_config_settings(app, settings: dict):
+    """Make sure expected config settings exist."""
+    missing_optional_config_settings = []
+    if "optional" in settings:
+        for setting in settings["optional"]:
+            if app.config.get(setting) is None:
+                missing_optional_config_settings.append(setting)
+        if missing_optional_config_settings:
+            warnings.warn(
```

Comment:
I wanted to use `app.logger.warn` here, but I can't set a warning category on that, so it felt like a downgrade and I ended up keeping `warnings.warn`.

Reply:
I'm resolving this discussion. Effectively I ended up using `app.logger.warning` anyways.